### PR TITLE
Add accordion to metadata form.

### PR DESCRIPTION
**Pull request by robgietema**
This PR add the feature to collapse and expand fieldsets in the metadata form in the sidebar. A helper method is also added which can be used to scroll to a specific field and focus on it.

### CODE REVIEW

**Comment by sneridagh** [comment]
@robgietema can we have some docs on how to use it?

**Comment by stevepiercy** [comment]
Would this be the appropriate location for developer docs for this feature? [Forms and widgets](https://6.docs.plone.org/volto/recipes/widget.html)

**Comment by robgietema** [comment]
I've added some docs in the location specified by Steve.

File: packages/volto/news/5760.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Ad accordion to metadata form. @robgietema
```
Comment (stevepiercy) [inline packages/volto/news/5760.feature:1]:
```suggestion
Add accordion to metadata form. @robgietema
```

File: docs/source/recipes/widget.md [hunk 2]
```diff
@@ -254,3 +254,15 @@ See [Storybook](https://6.docs.plone.org/storybook) with available widgets.
 ```{note}
 Please contribute to this section!
 ```
+
+## Sidebar
+
+In the edit form a sidebar is used when the form contains block data. There are some helper actions methods which can be used to change the form state.
```
Comment (stevepiercy) [inline docs/source/recipes/widget.md:260]: Avoid passive voice, prefer to address the reader. One sentence per line.

```suggestion
In the edit form, a sidebar is used when the form contains block data.
You can use the following helper action methods to change the form state.
```

File: docs/source/recipes/widget.md [hunk 3]
```diff
@@ -254,3 +254,15 @@ See [Storybook](https://6.docs.plone.org/storybook) with available widgets.
...
+
+### Setting tab
+
+You can use the `setSidebarTab` action to set the current active tab (metadata or block).
```
Comment (stevepiercy) [inline docs/source/recipes/widget.md:264]:
```suggestion
You can use the `setSidebarTab` action to set the current active tab, either via metadata or a block.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A few spelling, grammar, and style fixes. Thank you!

**Review by sneridagh — APPROVED** [review]
LGTM

**Review by stevepiercy — COMMENTED** [review]
Docs look good. Thank you!

**Pull request merged**